### PR TITLE
Update config and run command for dbsync (mainnet); closes #466

### LINKED ISSUE #466: Starting cardano-db-sync-extended errors with incorrect flags or key not found
**Describe the bug**
The documentation says to use --genesis-file as a flag. However cardano-db-sync-extended v3.1.0, v4.0.0 and master as of now() do not report this flag existing when running --help. Trying to use the flag in case it was undocumented shows:

`Invalid option '--genesis-file'`

Taking the flag and file out of the command and running it instead as:

```
~/git/cardano-db-sync# PGPASSFILE=$CNODE_HOME/priv/.pgpass cardano-db-sync-extended --config $CNO
DE_HOME/files/config.json --socket-path $CNODE_HOME/sockets/node0.socket --schema-dir schema/
```
returns the following error instead:

```
cardano-db-sync-extended: FatalError {fatalErrorMessage = "readDbSyncNodeConfig: Error parsing config: AesonException 
\"Error in $: key \\\"NetworkName\\\" not found\""}
```

**To Reproduce**
1. Run all the installation steps in [DBSync](https://cardano-community.github.io/guild-operators/#/Build/dbsync)
2. Run the steps in [start-cardano-db-sync-tool](https://cardano-community.github.io/guild-operators/#/Build/dbsync?id=start-cardano-db-sync-tool)
3. See error
4. Try command without `--genesis-file $CNODE_HOME/files/genesis.json`
5. See other error

**Expected behavior**
Expect command to start the cardano-db-sync-extended binary to have correct flags. When correct flags are used expect command to properly start cardano-db-sync-extended.

**Screenshots**
https://pastebin.com/NRm5y4h9

**Version:**
 - OS: Ubuntu 20.04
 - Product version: CNTools 5.4.0
 - Cardano Node version: 1.19.0
 - Network you're connecting to: 2017-09-23T21:44:51Z

### PR DESCRIPTION
**Pull request by rdlrt**
The changes were lose when moving to JSON

### CODE REVIEW

File: docs/Build/dbsync.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ scripts/postgresql-setup.sh --createdb
 ##### Start cardano-db-sync-tool
 ``` bash
 cd ~/git/cardano-db-sync
-PGPASSFILE=$CNODE_HOME/priv/.pgpass cardano-db-sync-extended --config $CNODE_HOME/files/config.json --genesis-file $CNODE_HOME/files/genesis.json --socket-path $CNODE_HOME/sockets/node0.socket --schema-dir schema/
+PGPASSFILE=$CNODE_HOME/priv/.pgpass cardano-db-sync-extended --config $CNODE_HOME/files/config.json --socket-path $CNODE_HOME/sockets/node0.socket --schema-dir schema/
```
Comment (TrevorBenson) [inline docs/Build/dbsync.md:49]: This modification will not take into consideration the associated changes to ptn0-mainnet.json as it still uses `--config $CNODE_HOME/files/config.json` when starting cardano-db-sync-extended.

Comment (rdlrt) [reply docs/Build/dbsync.md:49]: Prereqs.sh downloads relevant config (for mainnet: ptn0-mainnet.json) as config.json.

I am working on something for praos and other combinator network (hence, this PR only refers to mainnet)

**Comment by TrevorBenson** [comment]
Ahh yes, I had missed lines 239 and 241 and assumed it was the default
mainnet-config.json instead.

**Review by Scitz0 — APPROVED** [review]
LGTM

**Pull request merged**